src/snowcli/cli/__init__.py

```diff
@@ -152,6 +151,14 @@ def default(
         dir_okay=False,
         is_eager=True,
     ),
+    verbosity: LogVerbosity = typer.Option(
+        LogVerbosity.QUIET.value,
+        "--verbosity",
+        "-v",
```

Comment:
Can I do `snow -v cmd something ...` or do I have to specify some level?

Reply:
You can just without `-v` 😃, it has default value `quiet` (print only errors)

Reply:
Sorry if wasn't clear. If I want to enable verbosity can I pass just `-v` or do I have to provide some value?

Reply:
Changed to 2 options:

- `-v/--verbose` - print info and higher level logs
- `--debug` - print debug and higher level logs, logs have different format
- none of above - print only error logs